packages/VSNumberCompare/index.js

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, Text, StyleSheet } from 'react-native';
+
+const ZERO_PAD_LIMIT = 9;
```

Comment:
What are your thoughts of making these `defaultProps` instead of a global constant? Might make it a bit more reusable.